Makefile

```diff
@@ -37,4 +39,19 @@ clean:
 	mkdir -p .venv
 	cd .venv && $(.PYTHON3) -m venv --copies --prompt='[$(shell basename `pwd`)/.venv]' .
 
-	.venv/bin/python -m pip install --upgrade pip
\ No newline at end of file
+	.venv/bin/python -m pip install --upgrade pip
+
+
+download-jquery:
+	curl https://cdn.jsdelivr.net/npm/jquery@$(JQUERY_VERSION)/dist/jquery.min.js -o mara_app/static/jquery.min.js
+
+download-jquery-floatThead:
+	curl https://cdnjs.cloudflare.com/ajax/libs/floatthead/$(JQUERY_FLOATTHEAD_VERSION)/jquery.floatThead.min.js -o mara_app/static/jquery.floatThead.min.js
+
+download-bootstrap:
+	curl https://cdn.jsdelivr.net/npm/bootstrap@$(BOOTSTRAP_VERSION)/dist/css/bootstrap.min.css -o mara_app/static/bootstrap/bootstrap.min.css
+	curl https://cdn.jsdelivr.net/npm/bootstrap@$(BOOTSTRAP_VERSION)/dist/css/bootstrap.min.css.map -o mara_app/static/bootstrap/bootstrap.min.css.map
+	curl https://cdn.jsdelivr.net/npm/bootstrap@$(BOOTSTRAP_VERSION)/dist/js/bootstrap.min.js -o mara_app/static/bootstrap/bootstrap.min.js
+
+download-web-packages:
+	make -j upgrade-jquery upgrade-bootstrap
```

Comment:
This looks like it should be `make -j download-bootstrap download-jquery-floatThead 
download-jquery` ?